tests/unit/hubble/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jinaai/jina:latest
```

Comment:
this needs to use `test-pip` that is the one we use to test in CI

Reply:
changed